src/__stories__/_styles.scss

```diff
@@ -1,5 +1,6 @@
 @import "~@stoplight/tree-list/styles/_tree-list.scss";
-@import "~@stoplight/ui-kit/styles/_ui-kit.scss";
+@import "~@stoplight/mosaic/styles.css";
+@import "~@stoplight/mosaic/themes/default.css";
```

Comment:
> JSV doesn't have a CSS file at all? zipper_mouth_face

It may not. In fact, most of our projects have very lean stylesheets, or no stylesheets at all.
The idea was that we rely on Blueprint's styling as much as possible, and use tailwind for stuff like spacing.
Any custom styles we needed were added to ui-kit.

> This is weird, in this case we need to document that all callers should implement the same @imports, don't we @P0lip ?

Yup, true. I think it might be already documented.

Reply:
Yup, it does not. It's used internally, documenting might be good.